definitions/divorce/json/AuthorisationCaseState/AuthorisationCaseState-alternative-service-nonprod.json

```diff
@@ -41,6 +41,13 @@
     "UserRole": "citizen",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "03/02/2021",
```

Comment:
We should have had those permissions long time ago. They should be here as `boWelshReview` event can be triggered by systemupdate user role.

Systemupdate role didn't have enough permissions to trigger this event from pre-condition-state.

This bug didn't hit the prod yet.

It appeared when I fixed the way we execute unit tests. Unit test will save the world!